types/stpadoptions.pp

```diff
@@ -0,0 +1,13 @@
+# Matches Pad Options Keys And Values
+type Etherpad::Stpadoptions = Hash[Enum[
```

Comment:
I liked the struct more because you could set the data type per parameter. http://etherpad.org/doc/v1.6.6/#index_showlinenumbers tells me a lot of options are actually booleans so we can enforce that in the data type.

Reply:
Fixed.